src/lib.rs

```diff
@@ -147,6 +162,18 @@ pub const fn custom_integer(tag: u16, id: u32, value: u32) -> entry::IdAndInt {
     }
 }
 
+/// Create a 'Binary Info' entry containing a custom string entry.
```

Comment:
This should also be null-terminated, right?

Reply:
Absolutely,  thanks!

```suggestion
/// Create a 'Binary Info' entry containing a custom string entry.
///
/// The given string must be null-terminated, so put a `\0` at the end of
/// it.
```